libft/sources/map/mapss/mapss.h

```diff
@@ -31,7 +31,9 @@ typedef struct s_mapss
 int		ft_mapss_add(t_mapss *map, const char *key, const char *value);
```

Comment:
not alligned, not bloker, up to you

Reply:
what exactly?
![image](https://github.com/user-attachments/assets/a2a3d150-6146-4653-8d2c-17f4fc67ff41)

Reply:
norminette doesn't say anything...

Reply:
ok, i I based it on how it looks on GitHub, but you answered above that it has differrent fornt, you can resolve when read it